Installation.md

```diff
@@ -0,0 +1,259 @@
+## Setup ##
+
+### Notices before installation ###
+
+We assume you use [RVM](https://rvm.io/). If you don't, then replace `rvmsudo` with `sudo` during the installation process.
+
+:exclamation: RVM 1.17.x may have some [problems with rvmsudo](http://stackoverflow.com/questions/13765520/rvmsudo-command-not-working-properly).
+
+An X server is currently needed for browser tests to pass when using Selenium. `Xvfb` will do, but remember to set your `DISPLAY`.
+
+For testing environment PhantomJS (at least version 1.8.1) is required by Poltergeist gem to run headless browser tests. See [poltergeist](https://github.com/teampoltergeist/poltergeist) for documentation.
+
+If you are using Java 8, the following test will fail `RemoteSandboxForTesting should include tools.jar in the classpath for ant projects`. Please refer tp [issue #232](https://github.com/testmycode/tmc-server/issues/232) for further information.
```

Comment:
I would remove this comment from here - yes it fails until we fix it but anyways lets not have it in installation guide. What do you think @mpartel